site/_scss/site/objects/_components.scss

```diff
@@ -0,0 +1,149 @@
+@mixin breakpoint($point) {
```

Comment:
@amymanion this should be moved into a common location (such as `/site/_scss/site/utilities/_mixins.scss`) instead.